migrations/01_site-command_create_table_site_migration.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
Rename as specified in - https://github.com/EasyEngine/auth-command/pull/9#discussion_r215918902

Reply:
Ping when done